django_admin_index/conf.py

```diff
@@ -3,6 +3,9 @@
 from django.conf import settings as django_settings
 
 
+def default_dropdown_condition_function(parameter) -> bool:
+    return True
+
```

Comment:
Please move this to `django_admin_index.utils.should_display_dropdown_menu`, and change the implementation to contain the code that is now in the template:

```py
def should_display_dropdown_menu(request):
    return admin_index_settings.SHOW_MENU and request.user.is_authenticated and request.user.is_staff
```